### DISCUSSION TITLE
restore docker seq instance to windows?

### DISCUSSION THREAD

**Discussion by raymonvt** [datalust/seq-tickets#1321] [Help]
We currently have some seq instances running on docker but would like to migrate to windows. Is it possible to export the seq backup from docker and import it to windows? when i try this i get a error that the versions are diffrent

**Comment by nblumhardt** [comment]
Hi @raymonvt - thanks for getting in touch.

Which Seq versions are you running? If you're running the 2021.2.6459 container, you'll need to restore into a 2021.2.6459 Windows Seq install, and so on.

Migration from Docker to Windows this way isn't very frequently done, so there may be some gotchas along the way, but the basic backup/restore approach is the first one to try.

If running matched versions doesn't work for you, can you please let us know a bit more about your environment, and the kind of data/configuration you need to preserve?

Thanks,
Nick
